commands/next_lint_error.py

```diff
@@ -57,16 +57,25 @@ def _harvest_next(self) -> int:
             self.window.active_view().sel()[0].begin()
         )
         lines = set([])
+        next_change = None
```

Comment:
I don't think we need this variable to store to which line we should jump next, what we need is to check if `lines_list` is empty and return lines.pop() if it is empty